sphinxcontrib/jsontable/__init__.py

```
@@ -31,6 +31,14 @@ def setup(app: Sphinx) -> dict[str, Any]:
     """
     # Register the jsontable directive
     app.add_directive("jsontable", JsonTableDirective)
+    
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)